std/traits.d

```diff
@@ -3603,17 +3603,21 @@ private template hasUnsharedAliasingImpl(T)
     }
 }
 
-/**
- True if `S` or any type embedded directly in the representation of `S`
- defines an elaborate copy constructor. Elaborate copy constructors are
- introduced by defining `this(this)` for a `struct`.
+version (StdDdoc)
```

Comment:
Is it better to put the comment above the version statement, @adamdruppe?

Reply:
I think ddoc requires it inside the version since it doesn't support doc comments on the version. And this whole thing is just a hack for ddoc anyway since my doc generator handles it either way (in fact the version is pointless duplication there since it can directly document the alias and auto-link it too).

Reply:
Are you saying we should keep this as?

Reply:
Hmm could this change be cause of the ddoc errors at http://dtest.dlang.io/results/df6e6a1cc63f702fa20e106a116f317974ba6693/!base/?

Reply:
Well we should just drop ddoc entirely. But like since that's not happening today, yeah, just leave it  how it is.

Reply:
Ok, thanks.